src/components/ui/button.module.css

```diff
@@ -98,6 +98,10 @@ a.button {
   padding: 0.5rem;
 }
 
+.button.round {
+  border-radius: 100vw;
```

Comment:
Oh yeah, % amounts don't work well on rectangular containers, it gives you a weird ellipse, instead of a neat capsule, so I just default to vws and . It won't make a difference for perfectly round components, but I guess I just blocked % border radius from my head.

<img width="205" alt="Screenshot 2024-09-13 at 6 58 22 PM" src="https://github.com/user-attachments/assets/d1d48663-cb0c-4041-a511-d0b4a98365f2">